src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestBigFilesIT.java

```diff
@@ -40,8 +40,7 @@ public static Object[] compressionAlgorithms() {
     return new Object[] {"GZIP", "ZSTD"};
   }
 
-  @Parameter
-  public String compressionAlgorithm;
+  @Parameter public String compressionAlgorithm;
```

Comment:
could you run format.sh to see if this will be reverted back?

Reply:
`format.sh` suggested these changes.